### DISCUSSION TITLE
Infinite blinking

### DISCUSSION THREAD

**Discussion by sagamusix** [ZuluSCSI/ZuluSCSI-firmware#100] [Q&A] [answered]
I have a Korg Triton-Rack which was previously confirmed to work with the SCSI2SD 5.5 (see link below). I tried both my SCSI2SD 5.0 and a ZuluSCSI mini with it, but no luck. I have a strong feeling that there is an issue with the termination power, because connecting the USB cable to the SCSI2SD 5.0 makes it work. With the SCSI2SD 5.0 only on termination power, I see nothing happening at all, the device does not blink and it is not detected by the Triton-Rack. The ZuluSCSI mini is also not detected, but instead its LED just keeps blinking infinitely. The blinks are rather short and I have the impression that they are not as bright as the regular 1 / 3 / 5 blinks on "healthy" startup. Also, no log file is created at all.

Do you have any idea what these symptoms might mean? Supposedly the Triton-Rack [should provide enough termination power](http://www.korgforums.com/forum/phpbb2/viewtopic.php?t=121449) but what I'm seeing makes me think that this is not the case for me.

Edit: Worth noting is also that both units work if they are daisy-chained with another host inbetween that provides the power, which I believe confirms my theory even more that the termination power is the issue - but I really want to avoid this situation because the Triton-Rack *really* doesn't like it when the other device is turned on later than the Triton-Rack itself.)

**Comment by aperezbios** [comment, marked as answer]
@sagamusix it's quite possible that the 5 volt line on your Korg Triton is sagging, and not quite actually 5 volts. Your Korg Triton Rack is old enough to drink now, and capacitors from that era are known to have had many quality problems. They don't have to visibly leak to stop doing their job properly. They can also just dry out. You would need a volt meter/multimeter to check the 5V rail of the internal power supply in your Korg Triton. Do you have one? If not, maybe a friend does?

So, It isn't so much about your Korg not providing "enough power"....(the technically-correct term would be "current"), but very likely an issue with voltage sag, (below ~4.8 volts) which is incredibly common with older electronics. The solution is to replace antiquated capacitors, or simply work around the problem as you've done, by providing external power. 

Since it's not clear this is an issue to ZuluSCSI specifically, I'm going to convert this issue to a Discussion, which we can continue if you so desire.

**Reply by sagamusix** [reply]
Many thanks for the quick reply. I do have a multimeter so I can do some checks.

**Reply by sagamusix** [reply]
Sometimes YouTube is the best search engine... this person had exactly the same issue: https://www.youtube.com/watch?v=ZgPDM20r1Ps

**Reply by sagamusix** [reply]
I did some simple measurements from pin 25 to ground on the DB25 connector and indeed I got 4.7V or so on my Kurzweil K2500 (with which the ZuluSCSI works perfectly) but 0V on the Triton. So investigating that fuse as seen in that video might be my next step.
